source/frontend/src/assets/scss/colors.scss

```diff
@@ -1,23 +1,45 @@
 @import '@/../node_modules/@bcgov/design-tokens/css/variables.css';
 
+$pims-blue-20: #d9eaf7;
+$pims-blue-80: #1a5496;
+
 $pims-blue-50: #428bca;
-$pims-grey-80: #606060;
+
 $pims-yellow-10: #fef1d8;
 $pims-yellow-30: #fcba19;
 $pims-red-10: #f2dede;
+$pims-red-80: #a12622;
 $pims-white: #f2f2f2;
-$pims-blue-20: #d9eaf7;
 $pims-yellow-100: #6c4a00;
 
+$pims-grey-10: #e8ecef;
+
+$pims-grey-20: #959da5;
+$pims-grey-80: #606060;
+
+$pims-green-20: #dff0d8;
+$pims-green-80: #2e8540;
+
 $active-action-color: $pims-blue-50;
 $border-outline-color: $pims-grey-80;
 $warning-background-color: $pims-yellow-10;
 $number-background-color: $pims-yellow-30;
 $danger-background-color: $pims-red-10;
+$light-danger-color: $pims-red-10;
 $highlight-background-color: $pims-white;
 $filter-box-color: $pims-blue-20;
 $text-warning-color: $pims-yellow-100;
 $header-text-color: #{'var(--theme-blue-100)'};
 $header-border-color: #{'var(--theme-blue-90)'};
 $link-color: #{'var(--surface-color-primary-button-default)'};
 $link-hover-color: #{'var(--surface-color-primary-button-hover)'};
+
+$completed-color: $pims-green-80;
```

Comment:
some of the names here don't seem to match up with their usage in the app which I don't understand.

I think the idea behind these mappings are as follows:
1) in the upper section, define a list of colours **that are not already in the bcgov spec**
2) map those colors to subjective names within the pims app that give developers a clue where that colour is being used. ie. if pims has a "silly" button, perhaps we create a color called "silly-button-color" and map one of our non-gov colours to it. That way if we want to tweak the color of all silly buttons in the app we can do so in one place. (as a result, we may map the same colour to multiple concepts within pims).

Anyways, these colour subjective names seem to have been introduced specifically for these new status labels, and the status labels only, but these names don't seem to reflect that.

Reply:
I dont like the color names at all either. However, that's what they are called on figma, so I went for that for now.

Reply:
If ana went with that as a color name, it would likely be because she misunderstood the documentation I sent her during the color palette rework. She asked me for a list of old colors and new colors, and maybe got confused as to which was which (perhaps thinking that the list with more colors was newer, which is actually not the case). 

This is actually very unfortunate, because she may have chosen a number of colors from the list that we both (Ana and myself) agreed to remove from the application in favour of colors already in the bcgov spec.

We can go with this for now, and I'll log another tech debt item to clean up the colors again with Ana.